lib/images.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 
 # Image name and location
-IMAGE_OS=${IMAGE_OS:-Centos}
-if [[ "${IMAGE_OS}" == "Ubuntu" ]]; then
+IMAGE_OS=${IMAGE_OS:-centos}
+# NOTE: ${VAR,,} is making the varable lower case.
+# This is temporary and will be removed once we have changed IMAGE_OS to be
+# lower case in the CI.
+if [[ "${IMAGE_OS,,}" == "ubuntu" ]]; then
   export IMAGE_NAME=${IMAGE_NAME:-UBUNTU_20.04_NODE_IMAGE_K8S_${KUBERNETES_VERSION}.qcow2}
   export IMAGE_LOCATION=${IMAGE_LOCATION:-https://artifactory.nordix.org/artifactory/metal3/images/k8s_${KUBERNETES_VERSION}}
 elif [[ "${IMAGE_OS}" == "FCOS" ]]; then
```

Comment:
I'm not sure about these FCOS and FCOS-ISO. We do not use them in CI so we can keep them all upper case as they are acronyms.

Reply:
lets keep it for now